modules/core/src/main/java/org/apache/ignite/spi/discovery/tcp/internal/TcpDiscoveryNode.java

```diff
@@ -251,6 +253,22 @@ public void lastSuccessfulAddress(InetSocketAddress lastSuccessfulAddr) {
      */
     public void setAttributes(Map<String, Object> attrs) {
         this.attrs = U.sealMap(attrs);
+        Object id = this.attrs.get("CLUSTER_REGION_ID");
```

Comment:
CLUSTER_REGION_ID can only be set via attributes which is not convenient.

User should have an opportunity to set it:
1. via IgniteConfiguration.setClusterRegionId(long)
2. via system property - IgniteSystemProperties.IGNITE_CLUSTER_REGION_ID - which is very flexible and allows me to pass the ID even to ignite.sh
3. I would remove public attribute name.